style.css

```
@@ -102,3 +91,308 @@ p {
   align-items: center;
   margin: 0% 0% 0% 7%;
 }
+
+.headline-tagline {
+  height: 519px;
+  background-image: url("images/Group.png");
```

- [ ] YOu may consider using the correct path for your image.
```suggestion
  background-image: url("./images/Group.png");
```